kubernetes/chart/zulip/Chart.yaml

```diff
@@ -15,26 +15,26 @@ version: 0.4.0
 appVersion: "6.1-0"
 dependencies:
   - name: memcached
-    repository: https://charts.bitnami.com/bitnami
+    repository: https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami
     tags:
       - memcached
-    version: 6.0.16
+    version: 6.3.5
   - name: rabbitmq
-    repository: https://charts.bitnami.com/bitnami
+    repository: https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami
     tags:
       - rabbitmq
-    version: 8.32.0
+    version: 11.9.1
   - name: redis
-    repository: https://charts.bitnami.com/bitnami
+    repository: https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami
     tags:
       - redis
-    version: 16.8.7
+    version: 17.7.4
```

Comment:
For paper trail's sake, Bitnami claims this major version upgrade is fairly uninteresting unless Redis 7 is known to break your application: https://github.com/bitnami/charts/blob/main/bitnami/redis/README.md#to-1700